README.md

```diff
@@ -2,41 +2,28 @@
 
 This repo contains a smart contract making Pyth price feeds available on Neon EVM.
 
+It is **strongly recommended** to follow the [consumer best practices](https://docs.pyth.network/consumers/best-practices) when consuming Pyth data.
+
 ### How to use
 
-The Pyth wrapper supports both the Chainlink Aggregator Interface as well as the pyth-optimized native interface:
+To consume prices, you need to look up the price ID of the price feed for the symbol you're interested in. This corresponds to the account key of the Solana price account.
```

Comment:
document where to find this information. also looks like they need to translate the solana keys to hex?